tools/luthier.luau

```diff
@@ -417,10 +442,16 @@ local exitCode = 0
 
 if subcommand == "fetch" then
 	exitCode = fetchDependencies()
+elseif subcommand == "generate" then -- TODO cutesy name for generate
+	generateTypeDefinitionFiles()
+	generateStdLibFilesIfNeeded()
```

Comment:
Might be worth pulling this into a single generateAnyFiles function that we call first in all these cases?

Reply:
Would rather do this in another PR